recipes/tachyon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.99b6" %}
```

Comment:
Non-beta versions are old. Distributions like fedora, debian ship beta version because of this.